components/Layout/MsgForm/index.tsx

```diff
@@ -0,0 +1,91 @@
+import {FC} from 'react';
+import React, { useState } from 'react'
+
+
+export const Form:FC = ()=>{
+
+
```

Comment:
please add eslint and style your code to be aligned 
install prettier to format code